### PR TITLE
Add talOcc4 to CEGS_genomes

### PR DESCRIPTION
**Pull request by cadley-nyulangone**

### CODE REVIEW

File: dnase/trackhub/assets/CEGS_genomes [hunk 1]
```diff
@@ -1,4 +1,5 @@
 hg38
 mm10
 rn6
+talOcc4
```
Comment (mattmaurano) [inline dnase/trackhub/assets/CEGS_genomes:4]: john, this needs to be a diff vs. master. only this line should change

**Comment by mattmaurano** [comment]
john, can you back out the last commit and redo here rather than poening a new PR?

**Comment by cadley-nyulangone** [comment]
I see add_talOcc4 is restored to this pull request.  Is there something further I should do with it?

**Comment by mattmaurano** [comment]
Back out the changes to the other files, only CEGS_genomes  should change. The point is not to create extra PRs/branches.

**Comment by cadley-nyulangone** [comment]
I followed the rebase and merge instructions, and it looks like the master
in github got updated, which I did not think I had permission to do.

Before I touch this again, I'd like you to take a look at it.  The master
can probably get rolled back.  The process of undoing the commits and
rebasing the add_talOcc4 seems very complex.

**Pull request merged**